src/factors/Pose2D.jl

```diff
@@ -40,15 +40,38 @@ function preambleCache(dfg::AbstractDFG, vars::AbstractVector{<:DFGVariable}, pp
   (;manifold=M, ϵ0=getPointIdentity(M), Xc=zeros(3), q̂=getPointIdentity(M))
 end
 
+@inline function _vee(::SpecialEuclidean{2}, X::ArrayPartition{T, Tuple{SVector{2, T}, SMatrix{2, 2, T, 4}}}) where T<:Real
+  return SVector{3,T}(X.x[1][1],X.x[1][2],X.x[2][2])
```

Comment:
I just double checked sign on `X.x[2][2]` which looks good to me.

----

```julia
julia> A = [1 2;
       3 4]
2×2 Matrix{Int64}:
 1  2
 3  4

julia> A[2]
3
```

Reply:
not strictly a `vee`, since `X[2][2] = cos(theta)`

Reply:
Note the `X`, so the input should already be a tangent vector.